packages/main/src/themes/sap_fiori_3/Input-parameters.css

```diff
@@ -1,5 +1,14 @@
 @import "../base/Input-parameters.css";
 
 :root {
+	--_ui5_input_min_width: 2.75rem;
```

Comment:
Those styles are already imported from @import "../base/Input-parameters.css"; therefore if the value is the same (as it is), the code is redundant.